src/vs/base/browser/ui/list/listView.ts

```diff
@@ -1198,7 +1198,7 @@ export class ListView<T> implements IListView<T> {
 				while (e && !e.classList.contains('monaco-workbench')) {
 					e = e.parentElement;
 				}
-				return e || this.domNode.ownerDocument;
+				return e || this.domNode.ownerDocument.body;
```

Comment:
@bpasero this seems to have been lost when adopting aux windows. Its not possible to add a DOM element to the document so this would probably throw if it were ever to happen.

Reply:
Yes indeed! Also found another instance:

https://github.com/microsoft/vscode/blob/069fb11cd79c04b956b2c41df9264a0658d37556/src/vs/workbench/browser/parts/editor/breadcrumbsControl.ts#L197-L202